configure

```diff
@@ -681,6 +681,7 @@ def get_xcode_version(cc):
 
 def get_gas_version(cc):
   try:
+    os.environ["LC_ALL"] = "en_US"
```

Comment:
It would be slightly nicer to move this out of the try/except block and write it like this:
```python
env = os.environ.copy()
env['LC_ALL'] = 'en_US'
```
And then pass it with `subprocess.Popen(..., env=env)`.

Reply:
awesome! I was wondering how to prevent next command calls from being 'en_US' localized too. This fixes the side-effect.

Reply:
@oliveiraev Could you update the PR with this suggestion? :)

Reply:
Off course, I can!